style.css

```
@@ -0,0 +1,66 @@
+body {
+    margin: 0;
+    padding: 0;
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
+.header_back {
+    width: 100%;
+    height: 32px;
+    background: #1c2122;
+}
+
+.header_info {
+    color: #9bb3b8;
+}
+
+.wrapper {
+    width: 940px;
+    margin: 0 auto;
+}
+
+.info {
+    display: block;
+    margin-top: 6px;
+    float: left;
+}
+
+.info-item {
+    margin-right: 7px;
+    margin-left: 32px;
+}
+
+.info-item:first-child {
+    margin-left: 0;
+}
+
+.social {
+    float: right;
+    margin-top: 0;
+}
+
+.social_pic {
+    padding: 8px 15px 10px 15px;
+    border-left: solid 1px #9bb3b8;
+    font-size: 14px;
+}
+
+.social img {
+    border-right: solid 1px #9bb3b8;
+}
+
+.info a,
+.social a {
+    text-decoration: none;
+    color: inherit
+}
+
+ul {
+    list-style-type: none;
+    padding-inline-start: 0;
+    margin-block-end: 0;
```

```suggestion
    padding-inline-start: 0;
    margin-block-end: 0;
```
Это не стандартные свойства, используемые браузером. Используй стандартные для обнуления: `padding, margin`.

---

Done. Back to standard methods.